abdulfataiaka/react-nyne
# #160810998 Add tests to increase coverage

#### What does this PR do?
- Adds more tests suites and cases for implementations

#### Description of Task to be completed?
- Refactor main codes
- Refactor existing test codes
- Add more component test and helpers test

#### How should this be manually tested?
- Clone the repository
- Checkout int the branch `ft-implement-game-features-160810985`
- Install all dependencies
- Run `yarn run test` to see tests passing

#### What are the relevant pivotal tracker stories?
- [#160810998](https://www.pivotaltracker.com/story/show/160810998)

#### Screenshots (If Applicable)
